### PR TITLE
Fix: Margin missing sometimes in the PR sidebar

### LINKED ISSUE #4642: Margin missing sometimes in the PR sidebar
### Please ensure:

- [X] The bug is caused by Refined GitHub. It doesn't happen if I disable the extension.

### Description

- May be related to https://github.com/sindresorhus/refined-github/issues/4638

Steps _(it doesn't happen all the time, but reproducible enough)_

1. Starting in a issue, like here (maybe refresh this page first)
2. Click the Pull requests tab (ajaxed)
3. Open a PR (ajaxed)
4. Notice the lack of margin in the sidebar:
	Screen Shot 9
6. Refresh page
7. Notice it's fine
	Screen Shot

If it doesn't happen, just navigate between tabs and then enter a PR at last

### Example URL

https://github.com/sindresorhus/refined-github/pull/4626 but starting from another page

### Browser(s) used

Safari

### PR DESCRIPTION
**Pull request by cheap-glitch**
Fixes #4642 

## Test URLs
Steps to reproduce the bug:
 1. Go to repo home & reload the page
 2. Navigate to the PR tab
 3. Open a PR

## Screenshot
https://user-images.githubusercontent.com/46634000/135836816-0343d42c-110d-4289-91a1-cdb8faf1b265.mp4

### CODE REVIEW

File: source/features/clean-repo-sidebar.css [hunk 1]
```diff
@@ -9,6 +9,7 @@
 }
 
 /* Align the top of the sidebar with the main page content */
-.rgh-clean-repo-sidebar .gutter-condensed > :last-child {
```
Comment (cheap-glitch) [inline source/features/clean-repo-sidebar.css:12]: This selector was also affecting the sidebar in PR discussions

**Comment by fregante** [comment]
The change is reasonable, but the underlying issue probably still remains. Is the differing DOM GitHub’s fault or ours?

**Comment by fregante** [comment]
Checked: It's our fault. Try:

1. Disable RGH
2. Follow your screencast
3. Enable the extension (in Firefox everything will run immediately on the current page without refresh)
4. You see that this change is not necessary

Then try:

0. With RGH enabled
1. Follow your screencast
2. See bug

**Comment by cheap-glitch** [comment]
> The change is reasonable, but the underlying issue probably still remains.

The underlying "issue" is that the styles from `clean-repo-sidebar` stay loaded when navigating in ajax, so the affect a page they shouldn't (which is why the bug disappears upon refreshing the page).

**Comment by fregante** [comment]
D'oh, that's exactly what you did 😂 🙌

**Comment by fregante** [comment]
Love to see many bugfixes in the upcoming release, which will happen this week.

**Pull request merged**